e2e/framework/helper.go

```diff
@@ -460,6 +460,11 @@ func NewTestServers(t *testing.T, num int, conf func(*TestServerConfig)) []*Test
 		}
 	})
 
+	logsDir, err := initLogsDir(t)
```

Comment:
🤔 https://github.com/0xPolygon/polygon-edge/blob/develop/e2e/framework/ibft.go#L54

Reply:
There are two different e2e abstractions: one that deploys a chain of nodes with Ibft, and another one that deploys a dev chain. You added logs only for the first one (the link), this change adds logs for the second one.